src/ucp/core/ucp_am.c

```diff
@@ -583,7 +583,7 @@ static ucs_status_t ucp_am_contig_short(uct_pending_req_t *self)
     return ucp_am_short_handle_status_from_pending(req, status);
 }
 
-static ucs_status_t ucp_am_contig_short_reply(uct_pending_req_t *self)
+ucs_status_t ucp_am_contig_short_reply(uct_pending_req_t *self)
```

Comment:
not needed

Reply:
it's needed for `src/ucp/wireup/wireup.c`